expr/builtins/builtins.go

```diff
@@ -211,8 +211,29 @@ func Exists(ctx expr.EvalContext, item interface{}) (value.BoolValue, bool) {
 			return value.BoolValueTrue, true
 		}
 		return value.BoolValueFalse, true
-	case value.StringValue, value.BoolValue, value.NumberValue, value.StringsValue,
-		value.TimeValue, value.IntValue, value.SliceValue, value.MapIntValue:
+	case value.StringValue:
+		if node.Nil() {
+			return value.BoolValueFalse, true
+		}
+		return value.BoolValueTrue, true
+	case value.BoolValue:
+		return value.BoolValueTrue, true
+	case value.NumberValue:
+		if node.Nil() {
+			return value.BoolValueFalse, true
+		}
+		return value.BoolValueTrue, true
+	case value.IntValue:
+		if node.Nil() {
+			return value.BoolValueFalse, true
+		}
+		return value.BoolValueTrue, true
+	case value.TimeValue:
+		if node.Nil() {
+			return value.BoolValueFalse, true
+		}
+		return value.BoolValueTrue, true
```

Comment:
Again: why return the bool if it's always `true`? It doesn't seem like there's any possible failure case for an Exists check.